azure-devops/projects/selfcare-projects/provider.tf

```diff
@@ -9,14 +9,21 @@ terraform {
   required_providers {
     azuredevops = {
       source  = "microsoft/azuredevops"
-      version = "= 0.1.4"
+      version = ">= 0.1.4"
```

Comment:
maybe better use 0.1.8 version required for versioning

Reply:
Done